ftml/docs/Blocks.md

```diff
@@ -305,6 +308,20 @@ Some text __here!__
 [[/div]]
 ```
 
+### Equation Ref
+
+Output: `Element::EquationReference` / `<span>`
+
+Body: None
+
+Arguments:
+* None
+
+Example:
+```
+You can take the area of the circle[[eref Area-Circle]] and use it to find the object's volume.
+```
+
```

Comment:
I never realized this was a thing. This just shows a tooltip of the equation, right?

Reply:
Pretty much. It's like a footnote but not styled by major themes apparently lol